build.sh

```diff
@@ -88,40 +86,40 @@ BUILD_DIRS="${LIBCUGRAPH_BUILD_DIR}
 
 # Set defaults for vars modified by flags to this script
 VERBOSE_FLAG=""
-CMAKE_VERBOSE_OPTION=""
+CMAKE_VERBOSE_OPTION=()
 BUILD_TYPE=Release
-INSTALL_TARGET="--target install"
+INSTALL_TARGET=("--target" "install")
 BUILD_CPP_TESTS=ON
 BUILD_CPP_MG_TESTS=OFF
 BUILD_CPP_MTMG_TESTS=OFF
 BUILD_ALL_GPU_ARCH=0
-CMAKE_GENERATOR_OPTION="-G Ninja"
-PYTHON_ARGS_FOR_INSTALL="-m pip install --no-build-isolation --no-deps --config-settings rapidsai.disable-cuda=true"
+CMAKE_GENERATOR_OPTION=("-G" "Ninja")
+PYTHON_ARGS_FOR_INSTALL=("-m" "pip" "install" "--no-build-isolation" "--no-deps" "--config-settings" "rapidsai.disable-cuda=true")
```

Comment:
TIL that shellcheck turns strings with spaces into arrays. The potential issues it's trying to avoid don't seem to outweigh the clunkiness of using an array as a string with spaces IMO, but maybe I'll get used to it.

Reply:
That was probably a macro gone overboard by me.  I think I can strip out the quotes